tests/testimageattributes

```diff
@@ -0,0 +1,210 @@
+#! /bin/sh
```

Comment:
this shouldn't be checked in

Reply:
Oops.. I've removed this. Good news is that now everything looks like it works :D